src/Core/maps/src/Handlers/MapPin/MapPinHandler.Android.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Android.Gms.Maps;
+using Android.Gms.Maps.Model;
+using Microsoft.Maui.Handlers;
+
+namespace Microsoft.Maui.Maps.Handlers
+{
+	public partial class MapPinHandler : ElementHandler<IMapPin, MarkerOptions>
+	{
+		protected override MarkerOptions CreatePlatformElement() => new MarkerOptions();
+
+		public static void MapLocation(IMapPinHandler handler, IMapPin mapPin)
+		{
+			handler.PlatformView.SetPosition(new LatLng(mapPin.Location.Latitude, mapPin.Location.Longitude));
```

Comment:
Could we protect this in case Location is null?

Reply:
done